### ISSUE TITLE
header links direct me to 8ch.net not to my site

### ISSUE THREAD

**Issue by seelyowen** [OpenIB/OpenIB#296]
Most of the links at the top of my page are directing to 8ch.net not to my website. How do I edit the header bar to remove the 8ch.net links and direct them to my page instead.

**Comment by anoduck** [comment]
Yeah...It does that, and I have to admit, I am having difficulty myself finding exactly where to edit to change this behavior. Usually, it is in one of the files located in the template directory. But so far, have not found it.  

It is somewhere that is being called for page generation. If I find where, I will post the answer.

I am so going dedicate some time to writing a wiki page on how to install this chan engine. But, first I have to get it running myself.

**Comment by anoduck** [comment]
You need to edit your instance-config.php file and not your config.php file. 

After the middle of the instance-config.php file there is the 'boards' configuration setting. It will begin like this:
```

$config['boards'] = array(array('<i class="fa fa-home" title="Home"></i> home' => 'https://8ch.net', '<i class="fa fa-tags" title="Boards"></i> bo$
//$config['boards'] = array(array('<i class="fa fa-home" title="Home"></i>' => '/', '<i class="fa fa-tags" title="Boards"></i>' => '/boards.html', '<i$

```
Comment the first line and uncomment the second line. Edit the second to your needs and the particular navigational links you mentioned will redirect as you desire. 

This repo comes with a 'default' instance-config.php that has those directed to 8ch.net.
